### PR TITLE
refactor: to work with React@15.5+

### PR DESCRIPTION
**Pull request by benjycui**
ref: https://github.com/ant-design/ant-design/issues/5678

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,14 +41,15 @@
     "expect.js": "0.3.x",
     "pre-commit": "1.x",
     "rc-tools": "5.x",
-    "react": "0.14.x",
-    "react-addons-test-utils": "0.14.x",
-    "react-dom": "0.14.x"
+    "react": "^15.0.0",
+    "react-addons-test-utils": "^15.0.0",
```
Comment (afc163) [inline package.json]: react-addons-test-utils 替换为 react-dom/test-utils

**Comment by benjycui** [comment]
Updated

**Comment by benjycui** [comment]
```bash
➜  rate git:(refactor) npm owner ls rc-rate
afc163 <afc163@gmail.com>
yiminghe <yiminghe@gmail.com>
```

**Comment by afc163** [comment]
done

**Pull request merged**